diff --git a/vllm/platforms/cuda.py b/vllm/platforms/cuda.py
index 2025c41ab..395df4d1e 100644
--- a/vllm/platforms/cuda.py
+++ b/vllm/platforms/cuda.py
@@ -78,6 +78,7 @@ def _get_backend_priorities(
                 AttentionBackendEnum.FLASHMLA,
                 AttentionBackendEnum.FLASHINFER_MLA,
                 AttentionBackendEnum.TRITON_MLA,
+                AttentionBackendEnum.FA3_SPARSE_MLA,
                 AttentionBackendEnum.FLASHMLA_SPARSE,
             ]
     else:
